flocker/acceptance/testtools.py

```diff
@@ -215,9 +216,11 @@ class Node(PRecord):
     """
     A record of a cluster node.
 
-    :ivar bytes address: The IPv4 address of the node.
+    :ivar bytes address: The public address of the node.
+    :ivar bytes hostname: The address of the node, as reported by the API.
```

Comment:
This is a rather confusing set of attributes... Maybe `reported_address` instead of `hostname`?